src/org/labkey/test/selenium/LazyWebElement.java

```diff
@@ -39,6 +41,19 @@ public LazyWebElement(@NotNull Locator locator, @NotNull SearchContext searchCon
         _searchContext = searchContext;
     }
 
+    @Override
+    public boolean isDisplayed()
+    {
+        try
+        {
+            return super.isDisplayed();
+        }
+        catch (StaleElementReferenceException | NoSuchElementException ex)
+        {
```

Comment:
Would it be helpful to note in a log that this happened?

...or possibly, put a _staleReferenceCount member on the class and increment it, provide an isStale() method that returns whether or not the count is > 0?  Maybe start logging when it exceeds a threshold?